src/dashboard/components/Dashboard/Order/OrderList.tsx

```diff
@@ -0,0 +1,86 @@
+import { IOrder } from "@/types/order";
+import { DataGrid, GridActionsCellItem, GridColDef } from "@mui/x-data-grid";
+import EditIcon from "@mui/icons-material/Edit";
+import React from "react";
+import { useRouter } from "next/router";
+
+interface IOrderListProps {
+  orders: IOrder[];
+}
+
+export const OrderList = ({ orders }: IOrderListProps) => {
+  const router = useRouter();
+
+  const columns: GridColDef[] = [
+    {
+      field: "token",
+      headerName: "#",
+      editable: false,
+      flex: 1,
+      renderCell: (params) => {
+        return <span>{params.value}</span>;
+      },
+      minWidth: 300,
```

Comment:
I think that this property could help with rendering on smaller screens wouldn't it? So that it wouldn't crop the value but make whole table scrollable on X axis

Reply:
I tried that right now, so if you set minWidth on all columns, it behaves as expected (width of the column is the minimal possible), but there's still no horizontal scollbar. Therefore, IMO it's not much user-friendly